tests/test_sync_subclassof.py

```diff
@@ -0,0 +1,34 @@
+"""Tests for sync_subclassof.py"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+TEST_DIR = Path(os.path.abspath(os.path.dirname(__file__)))
+TEST_INPUT_DIR = TEST_DIR / 'input'
+TEST_OUTPUT_DIR = TEST_DIR / 'output'
+PROJECT_ROOT = TEST_DIR.parent
+sys.path.insert(0, str(PROJECT_ROOT))
+TEST_ONTOLOGY = TEST_INPUT_DIR / 'merged.owl'
+
+
+class TestSyncSubclassf(unittest.TestCase):
+    """TODO"""
+
+    def setUpClass(self):
+        """Set up"""
+        # TODO: convert to semsql if file not already present
+        #  - and add the .db file to the gitignore, maybe
+        # TODO: run pipeline
+        pass
+
+    def test_integration(self, use_cache=False):
```

Comment:
## Test incomplete @matentzn 
@matentzn Decide whether to merge now or pick solution.

<details><summary>Details</summary>
<p>

_[@hrshdhgd wrote](https://github.com/monarch-initiative/mondo-ingest/pull/363#issuecomment-1851111492)_:
> Is it wise to merge this without completing the test for it?

This is a good point. Basically we got stuck because I wanted to create a miniature version of `mondo.owl` that was adequate for testing, but Nico and I ended up kicking the can down the road.

However, there is still currently a miniature `mondo.owl` committed. So it should be possible to use that and write some tests. I think there would best be at least 2 tests to start, 1 for the "case 1" ('confirmed' axioms') and one for "case 3" ('added' axioms).

## Possible choices
a. Merge now, add tests later
b. Add tests now (tweak `mondo.owl` to make it smaller / more optimal)
c. Add tests now (no `mondo.owl` tweak

</p>
</details>

Reply:
TMD for me; feel free to make a call!

Reply:
I can't complete it by Thursday, so if we want to merge this by then, I'd say I'll do it after. I'd like to do this before ICD11 though.

Reply:
Will do later